office_space_allocation/fellow.py

```diff
@@ -0,0 +1,12 @@
+from office_space_allocation.person import Person
+
+
+class Fellow(Person):
+    def __eq__(self, other_person):
+        """
+        Tests for equality between this Person and the other_person
+        """
+        return (isinstance(other_person, Fellow) and
```

Comment:
I think it is necessary. By default, `==` comparisons are done using object ids. However, for my case, that can be erroneous especially after saving or loading state. That's why I overrode the `==` operator.